### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by HaruIjima-kun**
FIx error in HA 2024.11.0

### CODE REVIEW

File: custom_components/miio_yeelink/__init__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 from homeassistant import core, config_entries
 from homeassistant.const import *
-from homeassistant.config import DATA_CUSTOMIZE
+from homeassistant.core_config
```
Comment (lubosjerabek) [inline custom_components/miio_yeelink/__init__.py:11]: Needs to be `from homeassistant.core_config import DATA_CUSTOMIZE` instead

**Review by lubosjerabek — CHANGES_REQUESTED** [review]
Not sure if your change fixes the issue at all - suggested below

File: custom_components/miio_yeelink/__init__.py (hunk 1 above)
Comment (Chr1s16) [reply custom_components/miio_yeelink/__init__.py:11]: > @lubosjerabek followed your instructions and now it works again. Thanks!

**Review by pskaua — APPROVED** [review]
A simple change that restores the full functionality of the add-on

**Pull request closed without merging**